### PR TITLE
hw-mgmt: thermal: Add drivetemp and ibc sensors support to TC

### PR DESCRIPTION
**Pull request by sholeksandr**
1. Add drivetemp and ibc sensors support to TC
2. Add voltmon2 sensor to MSN2210 thermal data

Signed-off-by: Oleksandr Shamray <oleksandrs@nvidia.com>

### CODE REVIEW

**Comment by sholeksandr** [comment]
RERUN

**Comment by sholeksandr** [comment]
RERUN

File: usr/etc/hw-management-thermal/tc_config_msn5600.json [hunk 1]
```diff
@@ -42,7 +42,10 @@
...
 		"voltmon\\d+_temp": {"pwm_min": 30, "pwm_max": 100, "val_min": "!85000", "val_max": "!125000",  "poll_time": 60},
-		"sodimm\\d_temp" :{"pwm_min": 30, "pwm_max" : 70, "val_min": "!70000", "val_max": 95000, "poll_time": 60}
+		"sodimm\\d_temp" :{"pwm_min": 30, "pwm_max" : 70, "val_min": "!70000", "val_max": 95000, "poll_time": 60},
+		"drivetemp":      {"pwm_min": 30, "pwm_max": 70, "val_min": "!70000", "val_max": "!95000", "poll_time": 60},
+		"ibc\\d+":         {"pwm_min": 30, "pwm_max": 100, "val_min": "!80000", "val_max": "!110000", "poll_time": 60}
 	},
-	"sensor_list" : ["asic1", "cpu", "drwr1", "drwr2", "drwr3", "drwr4", "psu1", "psu2", "sensor_amb", "voltmon1", "voltmon2", "voltmon3", "voltmon4", "voltmon5", "voltmon6", "voltmon7", "voltmon8", "voltmon9", "voltmon10", "voltmon11"]
+	"sensor_list" : ["asic1", "cpu", "drivetemp", "drwr1", "drwr2", "drwr3", "drwr4", "ibc1", "ibc2", "ibc3", "ibc4", "psu1", "psu2", 
+	"sensor_amb", "sodimm1", "sodimm2", "voltmon1", "voltmon2", "voltmon3", "voltmon4", "voltmon5", "voltmon6", "voltmon7", "voltmon8", "voltmon9", "voltmon10", "voltmon11"]
```
Comment (ciju-nvidia) [inline usr/etc/hw-management-thermal/tc_config_msn5600.json:50]: Please remove this extra new line.

**Review by ciju-nvidia — APPROVED** [review]

**Comment by sholeksandr** [comment]
RERUN

**Comment by sholeksandr** [comment]
RERUN

**Pull request merged**